components/FaqPage/ImageGallery.tsx

```diff
@@ -0,0 +1,50 @@
+import Image from "next/image";
+import { ImageGalleryItem } from "@/lib/faq/faqImages";
+
+const DEFAULT_CAPTION_CLASSNAME = "font-spectral lowercase text-left font-medium tracking-tight text-[#4E4E4E] text-[16px] leading-[100%] md:text-[24px] ml-2 md:ml-4 italic antialiased";
+
+const getCaptionClasses = (captionClassName: string, caption: string) => {
+  return `${captionClassName} ${caption.includes("Start your process") ? "pr-10" : ""}`;
+};
```

Comment:
Single Responsibility Principle
- actual: multiple functions defined in the same file
- required: exactly one function is defined per file
I recommend you create a new lib in `/lib/faq/getCaptionClasses` for this function